config/migrate_test.go

```diff
@@ -1,80 +1,134 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-package config
+package config_test
 
 import (
+	"io/ioutil"
+	"os"
+	"path"
 	"testing"
 
+	"github.com/mattermost/mattermost-server/v5/config"
 	"github.com/mattermost/mattermost-server/v5/model"
-	"github.com/mattermost/mattermost-server/v5/testlib"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func getDsn(driver string, source string) string {
-	if driver == model.DATABASE_DRIVER_MYSQL {
-		return driver + "://" + source
+func TestMigrate(t *testing.T) {
+	files := []string{
+		"IdpCertificateFile",
+		"PublicCertificateFile",
+		"PrivateKeyFile",
+		"internal.crt",
+		"internal2.crt",
 	}
-	return source
-}
-
-func TestMigrateDatabaseToFile(t *testing.T) {
-	if testing.Short() {
-		t.SkipNow()
+	filesData := make([]string, len(files))
+	for i := range files {
+		filesData[i] = model.NewId()
```

Comment:
I understand that the `NewId()` call is just a way of creating random data for each file (please correct me if I'm wrong). If that's right, maybe we could add a comment stating that to make it clear.

Reply:
Yes, that's it! I'll clarify with a comment -- thanks :)